### PR TITLE
feat: display time intervals by default 

### PR DESCRIPTION
**Pull request by djayatillake**
### Reference:
Closes: #999

### Description:
makes time intervals default for timestamp and date fields unless specified 'OFF' in yaml

### Scope of the changes (select all that apply): 
- [ ] Backend  
- [ ] Docs

### CODE REVIEW

File: docs/docs/references/dimensions.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ All the properties you can customize:
 | type            | Dimension type        | |
 | description     | string                | |
 | sql             | string                | |
-| time_intervals  | `'default'` or string[] | `'default'` will be converted into `['DAY', 'WEEK', 'MONTH', 'YEAR']` for dates and `['MILLISECOND', 'DAY', 'WEEK', 'MONTH', 'YEAR']` for timestamps |
+| time_intervals  | `'default'` or string[] | `'default'` will be converted into `['DAY', 'WEEK', 'MONTH', 'YEAR']` for dates and `['RAW', 'DAY', 'WEEK', 'MONTH', 'YEAR']` for timestamps, as will not setting anything; if you want no time intervals set 'OFF' |
```
Comment (ZeRego) [inline docs/docs/references/dimensions.md:55]: Suggesting small changes to keep consistency.
```suggestion
| time_intervals  | `'default'` or `OFF` or string[] | `'default'` will be converted into `['DAY', 'WEEK', 'MONTH', 'YEAR']` for dates and `['RAW', 'DAY', 'WEEK', 'MONTH', 'YEAR']` for timestamps, as will not setting anything; if you want no time intervals set `'OFF'` |
```

File: packages/backend/src/dbt/translator.mock.ts [hunk 2]
```diff
@@ -65,6 +65,30 @@ const COLUMN_WITH_DEFAULT_TIME_INTERVALS: Record<string, DbtModelColumn> = {
     },
 };
 
+const COLUMN_WITH_NO_TIME_INTERVALS: Record<string, DbtModelColumn> = {
```
Comment (ZeRego) [inline packages/backend/src/dbt/translator.mock.ts:68]: I see you added the mocks for the new scenarios but no new tests in `/lightdash/packages/backend/src/dbt/translator.test.ts`

**Review by ZeRego — COMMENTED** [review]
This is looking great 💪  just left a couple of comments on improvments.

In `packages/backend/src/schema.json` Line 25 we should add 'OFF' as a possible value.
```
               "time_intervals": {
                    "oneOf": [
                        {
                            "const": "default"
                        },
                        {
                            "const": "OFF"
                        },
                        {
                            "type": "array",
                            "items": {
                                "type": "string"
                            }
                        }
                    ]
                }
```

File: packages/backend/src/dbt/translator.mock.ts (hunk 2 above)
Comment (djayatillake) [reply packages/backend/src/dbt/translator.mock.ts:68]: had a go at this

**Comment by djayatillake** [comment]
@ZeRego what does requesting a deployment mean and how do you make that happen? Is it a github action?

**Comment by djayatillake** [comment]
oh nice it actually opens an instance, how do you login? or what login should you use?

**Comment by ZeRego** [comment]
Thanks for the contributions 🎉 

e2e tests are passing here:
https://github.com/lightdash/lightdash/actions/runs/1607105268

**Pull request merged**